lessons/specifics/poolboy.md

```diff
@@ -57,7 +57,7 @@ We need to know little bit about the various configuration options in order to s
 
 `max_overflow:` maximum number of workers created if pool is empty. (optional)
 
-`strategy:` lifo or fifo, determines whether checked in workers should be placed first or last in the line of available workers. Default is lifo. (optional)
+`strategy:` :lifo or :fifo, determines whether checked in workers should be placed first or last in the line of available workers. Default is :lifo. (optional)
```

Comment:
@ayayalar could you make these: `:lifo` and `:fifo`?  Whenever we reference code we use ticks for syntax highlighting.